cmd/snap-exec/main.go

```diff
@@ -43,7 +44,7 @@ var opts struct {
 
 func main() {
 	if err := run(); err != nil {
-		fmt.Printf("cannot snap-exec: %s\n", err)
+		fmt.Fprintf(os.Stderr, "cannot snap-exec: %s\n", err)
```

Comment:
Do we need to do anything else special to prevent random stuff from interfering with tab completion?